src/core_plugins/kibana/public/home/components/add_data.less

```diff
@@ -13,3 +13,13 @@
 .addDataButton {
   line-height: normal;
 }
+
+.addDataFooterItem_highlight {
```

Comment:
Just wanted to double-check that EUI doesn't already have a generic / reusable style for these kinds of highlights.